squidpy/_constants/_constants.py

```diff
@@ -105,3 +105,10 @@ class RipleyStat(ModeEnum):
     F = "F"
     G = "G"
     L = "L"
+
+
+@unique
+class ScatterShape(ModeEnum):
```

Comment:
`class ScatterShape(ModeEnum) ->  class ScatterShape(str, ModeEnum)`

Reply:
it doesn't actually work

Reply:
In what sense? Running
```python
class X(str, sq._constants._constants.ModeEnum):
        foo = 'foo'
assert isinstance(X.foo, str)  # passed
```